src/renderer/components/atoms/button/types.ts

```diff
@@ -0,0 +1,34 @@
+import type {
+  ReactNode,
+  ButtonHTMLAttributes,
+  AnchorHTMLAttributes,
+} from 'react';
+
+import type { LinkProps } from 'react-router-dom';
+
+export type ButtonType = 'primary' | 'secondary' | 'warning';
+export type ButtonVariants = 'big' | 'small';
+
+export interface SimpleButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  className?: string;
+  children: ReactNode;
+  as?: ButtonType;
+  variant?: ButtonVariants;
```

Comment:
I think `children` and `className` can be inherited from the native interfaces in all cases. What do you think?

Reply:
Agree, even  if it exetends from some HTMLAttributes. Done!